server/systems/generic/models.js

```diff
@@ -5,51 +5,42 @@
 //----------------------------------------------------------------------------------------------------------------------
 
 import _ from 'lodash';
-import connect from 'thinky';
-
-import baseConfig from '../../../config';
 
 //----------------------------------------------------------------------------------------------------------------------
 
-var config = _.assign({}, baseConfig.rethink, { db: 'rpgk_generic' });
-
-var thinky = connect(config);
-var type = thinky.type;
-var r = thinky.r;
+import trivialModels from 'trivialmodels';
 
-var db = { r, type, errors: thinky.Errors };
+var types = trivialModels.types;
+var db = { errors: trivialModels.errors };
 
 //----------------------------------------------------------------------------------------------------------------------
 
-db.Character = thinky.createModel('characters', {
-    id: type.string(),
-    stats: type.array().default([]),
-    counters: type.array().schema(
-        type.object().schema({
-            name: type.string().required(),
-            value: type.number(),
-            min: type.number(),
-            max: type.number(),
-            steps: type.number()
-        }).removeExtra()
-    ).default([]),
-    rolls: type.array().schema(
-        type.object().schema({
-            name: type.string().required(),
-            expression: type.string()
-        }).removeExtra()
-    ).default([]),
-    notes: type.array().schema(
-        type.object().schema({
-            name: type.string().required(),
-            content: type.string()
-        }).removeExtra()
-    ).default([]),
-    user: type.string().required()
-});
+db.Character =  trivialModels.define(
+    {
+        name: 'Character',
+        driver: {
+            name: 'TrivialDB',
+            options: {
+                name: 'characters',
+                namespace: {
+                    name: 'generic',
+                    dbPath: 'server/db',
+                },
+                pk: 'id'
+            }
+        }
+    },
+    {
+        id: types.String({ pk: true }),
```

Comment:
Thought. Do we want to use `id`? The previous code did because RethinkDB wouldn't let us not, really. Buuut, we could call it `baseChar`as opposed to id.